resources/testdata/glyphs3/infinity.glyphs

```diff
@@ -28,5 +38,19 @@ width = 600;
 );
 }
 );
+metrics = (
+{
+type = descender;
+},
+{
+type = "cap height";
+},
+{
+type = "x-height";
+},
+{
+type = ascender;
+}
+);
```

Comment:
Do we now fail unless we have a metrics block? - would be nice to not have to modify infinity since this is not really the point of the test

Reply:
because the glyphs2 source includes all of these metrics, we need to either delete them there, or add them here; either way works for me.

What is the point of the test? Is it recorded anywhere?

Reply:
I fear it's documented only in the tests that use it. It's to test `glyphname = infinity;` which used to parse as a number with suboptimal results. For context, at the time we parsed a plist dag without any knowledge of the desired data type for fields. These days we have better intel about what output type we _want_.

Reply:
(note the lack of quotes)